src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -0,0 +1,606 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Collections.Generic
+{
+    [System.Runtime.InteropServices.ComVisible(false)]
+    [DebuggerTypeProxy(typeof(ICollectionDebugView<>))]
+    [DebuggerDisplay("Count = {Count}")]
+    public class LinkedList<T> : ICollection<T>, System.Collections.ICollection, IReadOnlyCollection<T>
+    {
+        // This LinkedList is a doubly-Linked circular list.
+        internal LinkedListNode<T> head;
+        internal int count;
+        internal int version;
+        private Object _syncRoot;
+
+        // names for serialization
+        const String VersionName = "Version";
```

Comment:
I believe these were used in GetObjectData and OnDeserialization, support for which was removed from this class.  I'll remove the fields.